_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -136,13 +136,19 @@ These examples demonstrate how to create variables in Python and Scala.
     <tr>
       <td class="python-block">
         <code>x = 1
-        <br>x = "Hi"</code>
+        <br>x = "Hi"
+        <br>x = """foo
```

Comment:
May I suggest to use a different name than `x` (e.g., `y`)?

Reply:
sounds good 👍🏻